### PR TITLE
TP-44 and mateba are blacklisted from fitting in backpacks

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Title, they are not fittable in bags anymore.

## Why It's Good For The Game
It was a funny run for the bag revolvers, it is time for them to be laid to rest with raid boss queen and xeno king in the realm of bad design memes.

## Changelog
:cl:
balance: TP-44 revolvers and Matebas no longer fit in bags
/:cl:

### CODE REVIEW

**Comment by SpaceLoveSs13** [comment]
No

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -13,6 +13,13 @@
 	max_storage_space = 24
 	var/worn_accessible = FALSE //whether you can access its content while worn on the back
 
+	cant_hold = list(
+		/obj/item/weapon/gun/revolver/standard_revolver,
+		obj/item/weapon/gun/revolver/mateba,
```
Comment (SpaceLoveSs13) [inline code/game/objects/items/storage/backpack.dm:18]: Wow

**Comment by Georgeur** [comment]
> No

provide reasoning or dont say anything at all?

File: code/game/objects/items/storage/backpack.dm (hunk 1 above)
Comment (Pariah919) [inline code/game/objects/items/storage/backpack.dm:18]:
```suggestion
		/obj/item/weapon/gun/revolver/mateba,
```

**Comment by Texicos** [comment]
the marines are backed into a corner, they are lashing out in anger

File: code/game/objects/items/storage/backpack.dm [hunk 2]
```diff
@@ -13,6 +13,13 @@
 	max_storage_space = 24
 	var/worn_accessible = FALSE //whether you can access its content while worn on the back
 
+	cant_hold = list(
+		/obj/item/weapon/gun/revolver/standard_revolver,
+		/obj/item/weapon/gun/revolver/mateba,
+		/obj/item/weapon/gun/revolver/m44,
+
```
Comment (Pariah919) [inline code/game/objects/items/storage/backpack.dm:20]:
```suggestion
```

**Comment by Tristan63** [comment]
clearly a salt pr from someone that's died to "Ocelot"

**Comment by SplinterGP** [comment]
> 
> 
> clearly a salt pr from someone that's died to "Ocelot"

ocelot himself told to nerf it

**Comment by SplinterGP** [comment]
can probably to the snub nose idea tomorrow, should be good rather than this and still allows to store revolver on satchel but with downsides

**Comment by DerrickChamberlain** [comment]
Yikes, how about we do not do that. I can send a picture of a revolver fitting in a backpack.

**Comment by Wayland-Smithy** [comment]
Yikes, how about we do. Want to bring realism into this?

Let's remove the staggetstun it applies instead, for big xenos at least. I can send a picture of the math of how much force a 44 handgun round imparts and how when it hits things the size of elephants their mass would absorb or deflect that energy failing to be "shaken by the sudden impact"

**Comment by TiviPlus** [comment]
but y whats wrong with meme

**Comment by Georgeur** [comment]
> but y whats wrong with meme


its not a meme, its optimal.

**Comment by SilencedMP5A5** [comment]
this is my nerf :)

**Comment by SplinterGP** [comment]
closing because i believe its not the proper way of nerfing infinite revolver pulling memes

**Pull request closed without merging**